packages/open-payments/src/client/index.ts

```diff
@@ -250,6 +262,16 @@ export const createUnauthenticatedClient = async (
   const { resourceServerOpenApi, walletAddressServerOpenApi, ...baseDeps } =
     await createUnauthenticatedDeps(args)
 
+  baseDeps.logger.debug(
+    {
+      validateResponses: !!args.validateResponses,
+      useHttp: baseDeps.useHttp,
+      requestTimeoutMs:
+        args?.requestTimeoutMs ?? config.DEFAULT_REQUEST_TIMEOUT_MS
```

Comment:
`args` is always defined

```suggestion
        args.requestTimeoutMs ?? config.DEFAULT_REQUEST_TIMEOUT_MS
```